api/src/main/java/org/apache/cloudstack/api/command/user/iso/DetachIsoCmd.java

```diff
@@ -48,6 +48,9 @@
             required=true, description="The ID of the virtual machine")
     protected Long virtualMachineId;
 
+    @Parameter(name=ApiConstants.FORCED, type=CommandType.BOOLEAN, description="If true, ejects the iso before detaching on vmware. Default: false", since = "4.15.1")
```

Comment:
```suggestion
    @Parameter(name = ApiConstants.FORCED, type = CommandType.BOOLEAN, description="If true, ejects the ISO before detaching on VMware. Default: false", since = "4.15.1")
```